### PR TITLE
GEOPY-481: fixup - protect against space in path

### PR DESCRIPTION
**Pull request by sebhmg**
**GEOPY-481 - Conflict with geosoft as default anaconda**

### CODE REVIEW

File: Start_applications.bat [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 setlocal EnableDelayedExpansion
 
-call %~dp0get_conda_exec.bat
+call "%~dp0get_conda_exec.bat"
 if !errorlevel! neq 0 (
   exit /B !errorlevel!
 )
 
-call !MY_CONDA_EXE! activate geoapps && jupyter notebook geoapps\index.ipynb
+set MY_CONDA=!MY_CONDA_EXE:"=!
```
Comment (andrewg-mira) [inline Start_applications.bat:9]: Is this line missing a matching quote " character?

**Review by domfournier — APPROVED** [review]
Looking good. I will test it.

File: Start_applications.bat (hunk 1 above)
Comment (domfournier) [reply Start_applications.bat:9]: Loooks like it, nice catch

Comment (sebhmg) [reply Start_applications.bat:9]: no, that is the syntax to remove existing quote in the path... before I wrap again the variable in quote when calling conda
(Do not ask my why, but conda fails if I just leave the quotes as is in the variable)

PS: did I mentioned already that I hate batch script?

Comment (andrewg-mira) [reply Start_applications.bat:9]: haha yes I actually wasn't sure. It definitely looks weird.

**Review by andrewg-mira — APPROVED** [review]

**Pull request merged**